packages/jest-message-util/src/index.js

```diff
@@ -46,6 +46,7 @@ type StackTraceOptions = {
 
 const PATH_NODE_MODULES = `${path.sep}node_modules${path.sep}`;
 const PATH_EXPECT_BUILD = `${path.sep}expect${path.sep}build${path.sep}`;
+const PATH_JEST_PACKAGES = `${path.sep}jest${path.sep}packages${path.sep}`;
```

Comment:
this will make actual errors in the jest code base harder to track down when developing.

If that's ok, we should delete line 48 as well, as it's covered by this (ref https://github.com/facebook/jest/pull/5094/files#r157363023)

Reply:
Ok. Let me know if I have the all clear to remove 48 & 49.

Reply:
If tests are still green, it'd be great to remove both. But 49 should cover what 48 does, so only 49 is needed